google-cloud-storage/src/main/java/com/google/cloud/storage/spi/v1/HttpStorageRpc.java

```diff
@@ -91,6 +101,13 @@ public HttpStorageRpc(StorageOptions options) {
     HttpTransport transport = transportOptions.getHttpTransportFactory().create();
     HttpRequestInitializer initializer = transportOptions.getHttpRequestInitializer(options);
     this.options = options;
+
+    // Open Census initialization
+    censusHttpModule = new CensusHttpModule(tracer, true);
+    initializer = censusHttpModule.getHttpRequestInitializer(initializer);
+    batchRequestInitializer = censusHttpModule.getHttpRequestInitializer(null);
```

Comment:
Two initializers. One is the default one which wraps the original initializer. The other one wraps nothing and will be used in batch request (see comments below in line 133).